src/main.cpp

```diff
@@ -29,6 +49,29 @@ int main() {
 			guild_config.close();
 
 			event.reply(dpp::message("Updated the role!").set_flags(dpp::m_ephemeral));
+		} else if (event.command.get_command_name() == "info") {
+			std::ifstream guild_config("../" + event.command.guild_id.str() + ".json");
+			dpp::snowflake role_id{};
+
+			if (guild_config.good()) {
+				json j;
+				guild_config >> j;
+				role_id = j["ROLE_ID"];
+			}
+
+			dpp::embed embed = dpp::embed()
+				.set_title("Bump Watcher - Information")
+				.set_footer(dpp::embed_footer{
+					.text = "Requested by " + event.command.usr.format_username(),
+					.icon_url = event.command.usr.get_avatar_url(),
+					.proxy_url = "",
+				})
+				.set_colour(dpp::colours::aqua)
+				.add_field("Bot Uptime", bot.uptime().to_string(), true)
+				.add_field("Memory Usage", std::to_string(rss() / 1024 / 1024) + "M", true)
+				.add_field("Role ID:", role_id ? std::to_string(role_id) : "Not set!", true);
```

Comment:
SIKE! That's about `dpp::snowflake`.
first of all, `dpp::snowflake` has `str()` which is basically `std::to_string(this)` or something like that. And then there's `role_id` which isn't clean (`!role_id.empty()`?)

Reply:
ah yeah,

btw, "isn't clean" is a little silly as it's a personal perference but I'm happy to write it as `.empty()` :)

Reply:
isn't clean